platform/android/java/src/org/godotengine/godot/xr/regular/RegularContextFactory.java

```diff
@@ -1,5 +1,5 @@
 /*************************************************************************/
-/*  PancakeContextFactory.java                                           */
+/*  RegularContextFactory.java                                           */
```

Comment:
If we rename from Pancake* to Regular*, maybe the files should be `git mv`ed too?

Reply:
They were `git mv` on my local client and I'm still seeing them as such. The github `files` dropdown also seems to show them as being moved, only the diff view makes it seem like it's still the same file with only internal changes.

Let me know if you're seeing otherwise, and I can look into it.

Reply:
Ah you're right, I was mislead as the path is so long that it's cropped in the GitHub view. The tooltip clarifies it:
![Screenshot_20190702_100905](https://user-images.githubusercontent.com/4701338/60495638-78b84f00-9cb1-11e9-8139-fb9d71834765.png)